src/routers/article_tag.js

```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
```

Comment:
@toluola Kindly refer to @Temmyogunbo suggested routing convention

Reply:
fixed